### ISSUE TITLE
Telescope Picker Complains about opts = nil

### ISSUE THREAD

**Issue by ei-eiron** [tomasky/bookmarks.nvim#16]
Enjoying my emoji bookmarks 💯 🚀 

Also when I try to use the telescope picker, I see:

E5108: Error executing lua ...scope.nvim/lua/telescope/previewers/buffer_previewer.lua:41: attempt to index local 'opts' (a nil value)
stack traceback:
	...scope.nvim/lua/telescope/previewers/buffer_previewer.lua:41: in function 'qflist_previewer'
	...y/bookmarks.nvim/lua/telescope/_extensions/bookmarks.lua:68: in function 'list'
	[string ":lua"]:1: in main chunk

Maybe there is something wrong with my mapping, or I am failing to pass in default opts somewhere? I'm in lazyvim btw, and fairly new to the plugin setup style
https://www.lazyvim.org/extras/util/project  

Notably, telescope opts is set to nil (?), but I'm not seeing this issue with other plugins. Let me know if I should send this to Telescope or Lazyvim..

Thanks 🤝 

---

```lua 
-- Overriding editor.lua for better grep with telescope, fzf
local Util = require("lazyvim.util")

return {

  {
    "nvim-telescope/telescope.nvim",
    config = function()
      local actions = require("telescope.actions")
      require("telescope").setup({
      defaults = {
        mappings = {
          i = {
            ["<esc>"] = actions.close,
            ["<c-j>"] = actions.move_selection_next,
            ["<c-k>"] = actions.move_selection_previous,
          },
        },
      },
      })
    end,
    dependencies = {
...
      {
        "tomasky/bookmarks.nvim",
        config = function()
          require("bookmarks").setup({
            -- sign_priority = 8,  --set bookmark sign priority to cover other sign
            save_file = vim.fn.expand("$HOME/.bookmarks"), -- bookmarks save file path
            keywords = {
              -- You can just copy these from emoji keyboard in browser (since it's using same font)
              ["@t"] = "🔥", -- mark annotation startswith @t ,signs this icon as `Todo`
              ["@w"] = "⚠️ ", -- mark annotation startswith @w ,signs this icon as `Warn`
              ["@f"] = "⛏ ", -- mark annotation startswith @f ,signs this icon as `Fix`
              ["@n"] = "🧠", -- mark annotation startswith @n ,signs this icon as `Note`
            },
            on_attach = function(bufnr)
              local keymap = vim.keymap.set
              local bm = require("bookmarks")
              keymap("n", "<leader>mm", bm.bookmark_toggle, { desc = "# Toggle Emoji Bookmark" })
              keymap("n", "<leader>mi", bm.bookmark_ann, { desc = "# Change annotation" })
              keymap("n", "<leader>mc", bm.bookmark_clean, { desc = "# Clean all marks in buffer" })
              keymap("n", "<leader>mn", bm.bookmark_next, { desc = "# Jump to next mark in buffer" })
              keymap("n", "<leader>mp", bm.bookmark_prev, { desc = "# Jump to prev mark in buffer" })
              keymap("n", "<leader>mq", bm.bookmark_list, { desc = "# Show marked file list in quickfix window" })
              require("telescope").load_extension("bookmarks")
              keymap(
                "n",
                "<leader>ml",
                [[:lua require("telescope").extensions.bookmarks.list()<cr>]],
                { desc = "# Telescope bookmarks list" }
              )
            end,
          })
        end,
      },
    },
...
```

**Comment by cybermelons** [comment]
I ran into this issue as well. Try this

>                 { desc = "# Telescope bookmarks list" }
This line should be
>                 { desc = "# Telescope bookmarks list", buffer = true }

**Comment by ei-eiron** [comment]
Interesting!
The error goes away, but also a picker does not appear (no messages)..
I'm assuming we want a command to work on its own regardless of the mapping options.

**Comment by cybermelons** [comment]
> Interesting! The error goes away, but also a picker does not appear (no messages).

Sorry, that wasn't a complete fix. I've made a PR to fix the telescope extension here: https://github.com/tomasky/bookmarks.nvim/pull/17

**Comment by Komar0ff** [comment]
> Sorry, that wasn't a complete fix. I've made a PR to fix the telescope extension here: #17

It works well only on the first screen, if you go over the bookmarks and bring up the telescope again, nothing works

**Comment by cybermelons** [comment]
> > Sorry, that wasn't a complete fix. I've made a PR to fix the telescope extension here: #17
> 
> It works well only on the first screen, if you go over the bookmarks and bring up the telescope again, nothing works

I've had issues with bookmarks disappearing. Unsure if it's related to your issue or if yours and mine are entirely new ones. I suggest opening a separate PR with more details on your issue

**Comment by ei-eiron** [comment]
Thanks @cybermelons @Komar0ff for your help. Agree it sounds like a new issue 🙌
